apps/prairielearn/src/tests/courseEditor.test.ts

```diff
@@ -139,48 +139,6 @@ const testEditData = [
       'courseInstances/Fa18/assessments/New_1/infoAssessment.json',
     ]),
   },
-  {
-    button: '.js-change-id-button',
-    formSelector: 'form[name="change-id-form"]',
-    data: {
-      id: 'newAssessment/nested',
-    },
-    action: 'change_id',
-    info: 'courseInstances/Fa18/assessments/newAssessment/nested/infoAssessment.json',
-    files: new Set([
-      'README.md',
-      'infoCourse.json',
-      'courseInstances/Fa18/infoCourseInstance.json',
-      'courseInstances/Fa18/assessments/HW1/infoAssessment.json',
-      'questions/test/question/info.json',
-      'questions/test/question/question.html',
-      'questions/test/question/server.py',
-      'courseInstances/Fa18/assessments/newAssessment/nested/infoAssessment.json',
-    ]),
-  },
-  // This second rename specifically tests the case where an existing assessment
-  // is renamed such that it leaves behind an empty directory. We want to make
-  // sure that that empty directory is cleaned up and not treated as an actual
-  // assessment during sync.
-  {
-    button: '.js-change-id-button',
-    formSelector: 'form[name="change-id-form"]',
-    data: {
-      id: 'newAssessmentNotNested',
-    },
-    action: 'change_id',
-    info: 'courseInstances/Fa18/assessments/newAssessmentNotNested/infoAssessment.json',
-    files: new Set([
-      'README.md',
-      'infoCourse.json',
-      'courseInstances/Fa18/infoCourseInstance.json',
-      'courseInstances/Fa18/assessments/HW1/infoAssessment.json',
-      'questions/test/question/info.json',
-      'questions/test/question/question.html',
-      'questions/test/question/server.py',
-      'courseInstances/Fa18/assessments/newAssessmentNotNested/infoAssessment.json',
-    ]),
-  },
```

Comment:
Removed these as the `change-id-button` has been removed from the assessment settings page. I added tests to `instructorAssessmentSettings.test` that test changing the path without the use of the `change-id-button`.